### ISSUE TITLE
Import of artifactory groups fails on 2.6.15

### ISSUE THREAD

**Issue by DaveTCode** [jfrog/terraform-provider-artifactory#196]
**Describe the bug**
Attempting to import artifactory groups into terraform state fails with an error like:
```
Error: 
400 GET https://artifactory.company.com/artifactory/api/security/groups/?includeUsers=false
{
  "errors" : [ {
    "status" : 400,
    "message" : "Bad Request"
  } ]
}
```

Notably there's a missing group name at the end of the GET request generated above.

The code to generate this was:
```
resource "artifactory_group" "test" {
  name             = "test"
  description      = "test"
  admin_privileges = false
  auto_join        = false
}
```

and the import command was `terraform import artifactory_group.test test`

Notably this same import works correctly on v2.3.10 AND running terraform plan work regardless of which version of the provider I pick providing the resource has already been imported.

**Requirements for and issue**
- [x] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue)
- [x] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`
  7.27.6
- [x] Your version of terraform
  0.14.5

**Expected behavior**
Resource to be imported

**Additional context**

I've never written a terraform provider before but I _think_ I might know what the issue is here. Maybe it's a good starting point to look at anyway.

https://learn.hashicorp.com/tutorials/terraform/provider-import?in=terraform/providers suggests that it is crucial for import that `resourceGroupRead` can fully derive it's object from `d.Id()` where in fact https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource_artifactory_group.go#L132 makes use of the group.Name property to populate the URL. In contrast https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource_artifactory_permission_target.go#L319 makes use of d.Id() to generate the permissions target read URL (and importing permission targets does work)

Maybe that's a trivial fix, maybe not! Sorry for not sending a PR, I'm not sure how to sensibly set up a test environment.

**Comment by chb0github** [comment]
Thanks. I think I know what the issue is

**Comment by chb0github** [comment]
You were dead right! Easy fix. I will merge this in on Monday after I train some people.

**Comment by chb0github** [comment]
fixed in 2.6.17
